resources/default_pipeline_environment.yml

```diff
@@ -209,8 +209,10 @@ steps:
       testOptions: 'specs'
   handlePipelineStepErrors:
     echoDetails: true
+    failOnError: true
     libraryDocumentationUrl: 'https://sap.github.io/jenkins-library/'
     libraryRepositoryUrl: 'https://github.com/SAP/jenkins-library/'
+    mandatorySteps: []
```

Comment:
maybe we can rename this to something like `stepsThatAlwaysFail`, just shorter 😉

Reply:
I think the positive notion is okay ...